File: How to write an SFTP file transfer client for WinForms/[VB]-How to write an SFTP file transfer client for WinForms/VB/Sftp/SftpSettingInfo.vb
Imports System.Text

Namespace ClientSample.Sftp
	Friend Class SftpSettingInfo
		#Region "SFTP"

		Public Const ServerOs As String = "ServerOs"

		#End Region

		Public Shared Sub SaveConfig(ByVal settings As SettingInfoBase)
'			#Region "SFTP"
			settings.SaveProperty(ServerOs)
'			#End Region
		End Sub

		Public Shared Sub LoadConfig(ByVal settings As SettingInfoBase)
'			#Region "SFTP"
			settings.LoadProperty(ServerOs, 0)
'			#End Region
		End Sub
	End Class
End Namespace

File: How to write an SFTP file transfer client for WinForms/[C#]-How to write an SFTP file transfer client for WinForms/C#/Sftp/SftpSettingInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ClientSample.Sftp
{
    class SftpSettingInfo
    {
        #region SFTP

        public const string ServerOs = "ServerOs";

        #endregion

        public static void SaveConfig(SettingInfoBase settings)
        {
            #region SFTP
            settings.SaveProperty(ServerOs);
            #endregion
        }

        public static void LoadConfig(SettingInfoBase settings)
        {
            #region SFTP
            settings.LoadProperty(ServerOs, 0);
            #endregion
        }
    }
}
